src/text/span.rs

```diff
@@ -275,6 +275,46 @@ impl<'a> Span<'a> {
             .filter(|g| *g != "\n")
             .map(move |g| StyledGrapheme { symbol: g, style })
     }
+
+    /// Converts this Span into a left-aligned [`Line`]
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # use ratatui::prelude::*;
+    /// let span = Span::styled("Test Content", Style::new().green().italic());
+    /// assert_eq!(span.left_aligned().alignment, Some(Alignment::Left));
```

Comment:
Consider the following perhaps as an example of how to use this effectively:

```suggestion
    /// assert_eq!(
    ///     "Test Content".green().italic().centered(),
    ///     Line::raw("Test Content")
    ///         .style((Color::Green, Modifier::Italic))
    ///         .alignment(Alignment::Center),
    /// );
```

Reply:
Hm, looks like this will produce different results: 
`Line { spans: [Span { content: "Test Content", style: Style { fg: Some(Green), bg: None, underline_color: None, add_modifier: ITALIC, sub_modifier: NONE } }], style: Style { fg: None, bg: None, underline_color: None, add_modifier: NONE, sub_modifier: NONE }, alignment: Some(Center) }`
and
`Line { spans: [Span { content: "Test Content", style: Style { fg: None, bg: None, underline_color: None, add_modifier: NONE, sub_modifier: NONE } }], style: Style { fg: Some(Green), bg: None, underline_color: None, add_modifier: ITALIC, sub_modifier: NONE }, alignment: Some(Center) }`
Does this matter? If not, I would just write an example without any assertions

Reply:
Yeah my bad, we recently changed line so that it had its own style. A styled span that is wrapped in an unstyled line is different to a styled line. I don't mind leaving the assertions off from this. I think it's pretty obvious what the result is based on the context.